### ISSUE TITLE
Range-And-Bearing errors when destroying the CesiumViewer

### ISSUE THREAD

**Issue by ldening** [articodeltd/angular-cesium#343]
**Intended outcome:**

Adding range-and-bearing component coupled with destroying the CesiumViewer should not throw errors / cause the application to crash. 

**Actual outcome:**

Have a component with an object for the CesiumViewer that incorporates range-and-bearing. The onDestroy function in the component calls viewer.destroy(). When navigating away from the page, the application will crash.  There are 2 different ways in which the application will crash.  

1. If the component is loaded, then navigated away from, the following error happens:
```
Error: Uncaught (in promise): TypeError: e[i].destroy is not a function
TypeError: e[i].destroy is not a function
at PrimitiveCollection.removeAll (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:1507790)
at PrimitiveCollection.destroy (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:1509443)
at PrimitiveCollection.removeAll (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:1507790)
at PrimitiveCollection.destroy (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:1509443)
at Scene.destroy (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:3027323)
at CesiumWidget.destroy (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:3266789)
at Viewer.destroy (https://www.angular-cesium.com/assets/cesium/Cesium.js:23:3367390)
at MapComponent.ngOnDestroy (https://angular-72iunz.stackblitz.io/~/src/app/map.component.ts:51:25)
at callProviderLifecycles (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:21563:22)
at callElementProvidersLifecycles (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:21531:17)

```

1. If the range-and-bearing functionality has been used to draw, and then the component is navigated away from, the error is:
```
Error: Uncaught (in promise): TypeError: Cannot read property 'getRealPositions' of undefined
TypeError: Cannot read property 'getRealPositions' of undefined
at PolylinesEditorService.getPositions (https://angular-72iunz.stackblitz.io/turbo_modules/angular-cesium@0.0.61/bundles/angular-cesium.umd.js:22024:29)
at BehaviorSubject.observableToExtend.dispose (https://angular-72iunz.stackblitz.io/turbo_modules/angular-cesium@0.0.61/bundles/angular-cesium.umd.js:21959:38)
at MapComponent.disposeRangeAndBearing (https://angular-72iunz.stackblitz.io/~/src/app/map.component.ts:45:22)
at MapComponent.ngOnDestroy (https://angular-72iunz.stackblitz.io/~/src/app/map.component.ts:49:14)
at callProviderLifecycles (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:21563:22)
at callElementProvidersLifecycles (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:21531:17)
at callLifecycleHooksChildrenFirst (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:21521:33)
at destroyView (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:29689:9)
at callWithDebugContext (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:30424:29)
at Object.debugDestroyView [as destroyView] (https://angular-72iunz.stackblitz.io/turbo_modules/@angular/core@8.2.14/bundles/core.umd.js:30132:16)
```

**How to reproduce the issue:**

Stackblitz: https://stackblitz.com/edit/angular-72iunz

**Version**
- angular-cesium@<version number>
"dependencies": {
    "@angular/common": "^8.0.0",
    "@angular/compiler": "^8.0.0",
    "@angular/core": "^8.0.0",
    "@angular/forms": "^8.0.0",
    "@angular/platform-browser": "^8.0.0",
    "@angular/platform-browser-dynamic": "^8.0.0",
    "@angular/router": "^8.0.0",
    "angular-cesium": "^0.0.61",
    "angular2parse": "1.0.10",
    "cesium": ">= 1.48.0",
    "core-js": "2",
    "rxjs": "^6.5.2",
    "tslib": "^1.10.0",
    "zone.js": "^0.9.1"
  },

**Comment by eitanfr** [comment]
Hi,
Thx for letting us know... we will check it soon

**Comment by eitanfr** [comment]
Hi, Fixed in #349 .
By the way, you don't need to call `dispose()` by yourself if the component is destroyed, the editor handles it by himself.  
Same for the routing, if the route change cesium is removed from the DOM...
